Updated troubleshooting.md with recommendation of using args for heroku (#5197)
puppeteer/puppeteer: docs/troubleshooting.md

The url for the buildpack is https://github.com/jontewks/puppeteer-heroku-buildpack

> Ensure that you're using `'--no-sandbox'` mode when launching pupeteer. You can achieve it by passing it as an argument to your `.launch()` call, like this `puppeteer.launch({args: ['--no-sandbox']});`

Reviewer: ```suggestion
Ensure that you're using `'--no-sandbox'` mode when launching Puppeteer. This can be done by passing it as an argument to your `.launch()` call: `puppeteer.launch({ args: ['--no-sandbox'] });`.
```

Author: Thank you for your suggestions! And sorry for the typos.
